docs/src/manual.md

```diff
@@ -529,6 +529,19 @@ julia> suite
 As you might imagine, `BenchmarkGroup` supports a subset of Julia's `Associative` interface. A full list of
 these supported functions can be found [in the reference document](reference.md#benchmarkgrouptagsvector-datadict).
 
+One can also create a nested `BenchmarkGroup` simply by passing the keys, similar to how
+`mkdir -p` works:
```

Comment:
It's a bit niche, not sure that precision helps a lot of users. For instance I know what `mkdir` does but I've never heard about the `-p` flag.

Reply:
Ah, good point. Seems like I’ve spent so much time in a unix terminal that I managed to convince myself this was common knowledge 😂

Reply:
Fixed